src/pgrn-custom-scan.c

```diff
@@ -148,14 +142,12 @@ PGrnOpInOpfamily(Relation index, OpExpr *opexpr)
 }
 
 static void
-PGrnScanIndexDataInit(Relation index, List *quals, PGrnScanIndexData *data)
+PGrnTraverseQuals(Relation index, List *quals, List **scanKeySources)
```

Comment:
Can we use better name?
This doesn't just traverse quals, right?
This collects scan key sources from quals, right?